backend/src/controllers/coordinatorController.js

```
@@ -66,29 +89,55 @@ const getAllOrganizations = async (req, res) => {
   }
 };
 
-const getAllTeams = async (req, res) => {
+const getTeamsByDisasterId = async (req, res) => {
   try {
+    const { disaster_id } = req.params; 
     const { page = 1, limit = 10 } = req.query;
     const offset = (parseInt(page) - 1) * parseInt(limit);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)